manager/container.go

```diff
@@ -342,10 +343,10 @@ func (c *containerData) housekeeping() {
 
 		// TODO(vmarmol): Export metrics.
 		// Run custom collectors.
-		nextCollectionTime, _, err := c.collectorManager.Collect()
-		if err != nil && c.allowErrorLogging() {
-			glog.Warningf("[%s] Collection failed: %v", c.info.Name, err)
-		}
+		// nextCollectionTime, _, err := c.collectorManager.Collect()
+		// if err != nil && c.allowErrorLogging() {
+		//	glog.Warningf("[%s] Collection failed: %v", c.info.Name, err)
+		//}
 
 		// Next housekeeping is the first of the stats or the custom collector's housekeeping.
```

Comment:
Do we need to retain the metric collection here, @rjnagal , @vmarmol ? Only issue is that the metric collection time will not be included in housekeeping time in this case. 

However, the current approach of pushing metric collection to updateStats seems more convoluted.